.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea
 .vscode
+.DS_Store
```

Comment:
Yes, it's the files that Mac added.

Reply:
👋 I know that we all use Macs, but it's not _technically_ a requirement that you do so. As such, developer-specific preferences like operating system and IDE should be managed using a [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) rather than doing so per-repository.

This repository is admittedly a little different since it's public, so making it easier for external contributors to do the right thing is appreciated, but you should still take my suggestion for the sake of our internal repositories.

Reply:
Thanks for the recommendation 🙇